runners/spark/pom.xml

```diff
@@ -128,6 +128,83 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>2.2.0</version>
```

Comment:
How about adding hadoop.version property and using it for hadoop-mapreduce-client-core as well ?